.drone.yml

```diff
@@ -116,6 +116,10 @@ steps:
   # Also fetch the target branch (which is staging or staging-next for pull requests.) We need this for determining which tests to run based on changed files.
   - git remote set-branches --add origin $DRONE_TARGET_BRANCH
   - git fetch --depth 100 origin $DRONE_TARGET_BRANCH
+  # Always fetch staging branch because the change detection logic compares against the staging branch even when the base
+  # branch is staging-next:
+  # https://github.com/code-dot-org/code-dot-org/blob/8ea6ee6b63abd36e51c69d400eefdaa87154030a/lib/cdo/git_utils.rb#L104-L113
+  - git fetch --depth 100 origin staging
```

Comment:
why is it necessary to fetch to a depth?

Reply:
I don't see why it's necessary.  My intent was to preserve the existing logic that fetches the target branch with a depth of 100 commits.

Reply:
Ah, it's needed to support the i18n sync process? https://github.com/code-dot-org/code-dot-org/commit/37120d86d31ff5acbbe2f5830f0b02359c4ba959

Reply:
We specify a depth to do a shallow clone, which is faster. We need a high enough depth so that the commit where the feature branch was cut from the base branch is included (I don't remember exactly which step needs this). The whole thing probably could be improved, haven't looked at it since early last year.

Reply:
that's why it's necessary to fetch to a depth for the feature branch, but if we're just fetching the staging branch for the sake of change detection, shouldn't we only need to fetch the latest commit?

Reply:
I'm not sure (don't know what the git command being run is), good point though. Worth testing.